src/api.js

```diff
@@ -0,0 +1,19 @@
+import axios from "axios";
+
+const BASE_URL = "https://burhan-s-project.onrender.com/api";
+
+export const fetchAllArticles = () => {
+  return axios.get(`${BASE_URL}/articles`).then((res) => res.data.articles);
+};
+
+export const fetchSingleArticle = (article_id) => {
+  return axios
+    .get(`${BASE_URL}/articles/${article_id}`)
+    .then((res) => res.data.article[0]);
+};
+
+export const fetchArticleComments = (article_id) => {
+  return axios
+    .get(`${BASE_URL}/articles/${article_id}/comments`)
+    .then((res) => res.data.articleComments);
+};
```

Comment:
I would recommend having a read of the following notes on the recommended way to do this with Axios:

l2c.northcoders.com/courses/sd-notes/front-end/#sectionId=axios,step=creating-an-axios-instance